Dev/src/components/TeacherInfoEntries.js

```diff
@@ -161,4 +165,11 @@ const styles = StyleSheet.create({
         borderBottomColor: colors.grey,
         borderBottomWidth: 0.25
     },
+    infoTextInput: {
```

Comment:
Maybe we should use our fontStyles instead

Reply:
This is the container style.